### ISSUE TITLE
Inconsistent Casing in Headings 

### ISSUE THREAD

**Issue by brishtibheja** [ankitects/anki-manual#259]
Parts of the docs are using sentence casing in headings, including but not limited to Searching and Getting Help.

**Comment by Danika-Dakika** [comment]
I don't think this is actionable as an issue. Before anyone could take it up, there would need to be an inventory of what you are asking to have changed -- or at least your position on what the style should be, so that can be discussed. When you do that, please also clarify which headings/levels of headings you're talking about.

**Comment by brishtibheja** [comment]
We use Title cases in Headings most of the time. So the sentence casing needs to change. This was a more "note to myself" yes.
